modules/serverless-ossm-external-certs.adoc

```diff
@@ -11,9 +11,9 @@ By default, the {SMProductShortName} mTLS feature only secures traffic inside of
 .Prerequisites
 
 * You have cluster administrator permissions on {ocp-product-title}, or you have cluster or dedicated administrator permissions on {rosa-product-title} or {dedicated-product-title}.
-
 * You have installed the {ServerlessOperatorName} and Knative Serving.
-* Install the OpenShift CLI (`oc`).
+* You have installed the OpenShift CLI (`oc`).
+* You have the `knative-serving-ingress` namespace, or you have installed the {ServerlessOperatorName}, which automatically creates this namespace.
```

Comment:
"You have the `knative-serving-ingress` namespace" - This is a little unclear to me, can we rephrase it a little?